content/documentation/pages/5-feature-guides/2-batch/4-monitoring.md

```diff
@@ -218,3 +224,89 @@ applications:
 services:
   - mysql
 ```
+
+## Kubernetes
+
+This section describes how to set up Prometheus and InfluxDB for Kubernetes.
+
+### Prometheus
+
+Prometheus is a popular pull-based time series database that pulls metrics from the target applications from a pre-configured endpoint.
+Data Flow leverages the [Prometheus RSocket](https://github.com/micrometer-metrics/prometheus-rsocket-proxy) to establish a persistent, bidirectional connections between all Task applications and one or more `Prometheus RSocket Proxy` instances. Prometheus is configured to scrape each proxy instance. Proxies in turn use the connection to pull metrics from each Task application. The scraped task metrics are viewable through Grafana dashboards.
+
+Out of the box, each binder middleware configuration file defines attributes to enable metrics and their supporting properties. You can find settings in: `src/kubernetes/server/server-config.yaml`. The main point of interest is the following configuration section:
+
+```yaml
+applicationProperties:
+  task:
+    management:
+      metrics:
+        export:
+          prometheus:
+            enabled: true
+            rsocket:
+              enabled: true
+              host: prometheus-proxy
+              port: 7001
+grafana-info:
+  url: 'https://grafana:3000'
+```
+
+In this configuration, Prometheus metrics are enabled and configured to send to a prometheus rsocket proxy.
+
+With Prometheus, Grafana, Spring Cloud Data Flow, and any other services as defined in the [Getting Started - Kubernetes](%currentPath%/installation/kubernetes) section up and running, you are ready to collect metrics.
+
+<!--IMPORTANT-->
+
+The address used to access the Grafana UI depends on the Kubernetes platform the system is deployed to. If you are using (for example) GKE, the load balancer address would be used. If using Minikube (which does not provide a load balancer implementation), the IP of the Minikube (along with an assigned port) is used. In the following examples, for simplicity, we use Minikube.
+
+<!--END_IMPORTANT-->
+
+To obtain the URL of the Grafana UI when it is deployed to Minikube, run the following command:
+
+```bash
+$ minikube service --url grafana
+http://192.168.99.100:31595
+```
+
+In the preceding example, you can reach the Grafana dashboard at http://192.168.99.100:31595. The default credentials are as follows:
+
+- User name: admin
+- Password: password
+
+The Grafana instance is pre-provisioned with two dashboards:
+
+- Tasks: http://192.168.99.100:31595/d/scdf-tasks/tasks?refresh=10s
+
+You can collect metrics on a per-task, per-batch basis or apply metrics collection to all deployed applications globally.
+
+To deploy a custom Task application (`task-demo-metrics`) and define two tasks (`task1` and `task2`):
+
+```bash
+dataflow:>app register --name myTask --type task --uri docker://tzolov/task-demo-metrics:latest
```

Comment:
Could we move it to springcloud or wherever else the microsite images are?

Reply:
I don't know any other dockerhub space but `springcloud`.  Are there dedicated microsite images somewhere?

Reply:
For the time being i've pushed it to `docker://springcloud/task-demo-metrics:latest`